.gitignore

```diff
@@ -33,4 +33,39 @@ out/
 /.nb-gradle/
 
 ### VS Code ###
-.vscode/
\ No newline at end of file
+.vscode/
+
+# dependencies
+/frontend/node_modules
```

Comment:
Denne kan skrives som kun `node_modules` da det aldri vil være behov for en slik mappe i backend uansett.